### ISSUE TITLE
show the search result

### ISSUE THREAD

**Issue by tiantangbuyaowo** [notepadqq/notepadqq#963]
nodepad++ is a good software,it has a view for show the search result,it can show all the search result,but notepadqq can not do it,

**Comment by rolandjon** [comment]
Yes, this is a very important feature. Does anyone care about this?
